doc/development/plugins.rst

```diff
@@ -44,13 +44,14 @@ Creating your device
     returns correct expectation values.
 
 The first step in creating your PennyLane plugin is to create your device class.
-This is as simple as importing the abstract base class :class:`~.Device` from PennyLane, and subclassing it:
+This is as simple as importing the abstract base class :class:`~.QubitDevice` from PennyLane,
```

Comment:
We expect here that the developer wants to develop a qubit plugin. Should that not be mentioned that this is a special case?

Reply:
Maybe also mention why `Device.name` etc are attributes of the parent class...

Reply:
I think this is a very good point! (I just realized that I also took it for granted that it should be a qubit based device for sure!)

Reply:
What if there was a new page with the ``Device`` specific instructions which could then be updated once ``CVDevice`` is up and running?

Then a note here would suffice for people interested in building a ``CV`` based ``Device`` in the meantime

Reply:
Good idea!

Reply:
Will this make the TOC too long though? We are currently in the 'sweet spot'

Reply:
Would it be possible to have the main page stay like this but have a separate one that wouldn't appear on the left bar but could be accessed from this page? That way neither the TOC would become longer nor would the PL sidebar have more items on the left.

Reply:
I'd make it simple:  Just state here "We will demonstrate this with qubit-based devices inheriting from the QubitDevice class" or so. 

Either delete the "Supporting new CV operations" section altogether, turn its main message into a note box explaining more info is coming up, or just add a sentence pointing out why you talk about CV now.

In future, I'd make two subsections "Qubit" and "CV" even though it gets long. This will be read by only very few people who use it like a manual, it can be as long as it needs to IMO.

Reply:
Added this:
```rest
.. important::

    PennyLane supports both qubit and continuous-variable (CV) devices. However, from
    here onwards, we will demonstrate plugin development focusing on qubit-based devices
    inheriting from the :class:`~.QubitDevice` class.
```
Thanks @mariaschuld!